Replace "omim" with "disease" in case page query search and related functions (#4459)
Clinical-Genomics/scout: CHANGELOG.md

- Maintain Matchmaker Exchange and Beacon submission status when a case is re-uploaded
- Inheritance mode from ORPHA should not be confounded with the OMIM inheritance model
- Decipher link URL changes

> - Refactored casepage to use "disease" instead of "omim" where orpha-terms are also handled

Reviewer: ```suggestion
- Refactored code in cases blueprints to use "disease" instead of "omim" to encompass also ORPHA terms
```
